cypress/integration/pages/search_spec.ts

```diff
@@ -0,0 +1,18 @@
+/// <reference types="cypress" />
+export {};
+
+describe('Test Search Page', () => {
+  beforeEach(() => {
+    cy.visit('/search');
+  });
+  it('has a search form', () => {
+    cy.get('form').type('world population').should('contain', 'Search');
+  });
+  it('should return a search result', () => {
+    cy.get('form').find('[type="text"]').type('world population');
+    cy.get('form').submit();
+    cy.url().should('include', 'search?q=world%20population&sort=');
+    cy.get('.text-3xl').should('have.text', '1 results found');
```

Comment:
Again, if I go and publish another dataset for that query, this test would fail.